python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -245,6 +249,19 @@ def optimize(self, model: nn.Module,
                | grid mode. We will sort and combine according to the value you specified to
                | get the search range.
 
+        :param dynamic_axes: dict or boolean, default to True. By default the exported onnx model
+               will have the first dim of each input as a dynamic batch_size. If dynamic_axes=False,
```

Comment:
Should we modify the description for the `by default` case to include the logic added [here](https://github.com/intel-analytics/BigDL/pull/6869)?

Reply:
Done, have changed to `each Tensor input`